### DISCUSSION TITLE
ZepetoPlayer 카메라 조정

### DISCUSSION THREAD

**Discussion by Obsihill** [naverz/zepeto-studio-kor#1219] [5. Q&A]
<img width="389" alt="image" src="https://user-images.githubusercontent.com/106140401/186341250-477339eb-5710-4fea-9dcb-7ba378d6c758.png">

플레이도중에 카메라 상하 최대값 변경은 불가능한가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 네 현재는 세팅값이므로 런타임 제어는 지원하고 있지 않습니다.
